src/components/thumbnail/tab-panel-documents-section.tsx

```diff
@@ -83,21 +83,24 @@ export const TabPanelDocumentsSection = observer(({ tab, section, index, numOfSe
                                   onSelectNewDocument, onSelectDocument, onDocumentDragStart,
                                   onDocumentStarClick, onDocumentDeleteClick }: IProps) => {
     const { user } = stores;
-    const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument;
+    const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument
+                                      && section.type !== ENavTabSectionType.kProblemDocuments;
     const newDocumentLabel = getNewDocumentLabel(section, stores);
-    const numPanels = user.isNetworkedTeacher ? numOfSections + 1 : numOfSections;
-    const isTopPanel = index === 0 && numPanels > 1;
-    const isBottomPanel = index === numOfSections - 1 && index > 0;
-    const tabName = tab.toLowerCase().replace(' ', '-');
+    const numPanels = numOfSections && user.isNetworkedTeacher ? numOfSections + 1 : numOfSections;
+    const isTopPanel = (numOfSections && numPanels) && index === 0 && numPanels > 1;
```

Comment:
```suggestion
    const isTopPanel = numOfSections && index === 0 && numPanels > 1;
```
you don't need to test `numPanels` twice.